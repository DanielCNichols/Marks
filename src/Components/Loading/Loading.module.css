.loading {
  position: absolute;
  width: 50%;
  top: 50%;
  left: 50%;
  transform: translateX(-50%) translateY(-50%);
  font-size: 6vw;
  text-align: center;
}

@media screen and (min-width: 450px) {
  .loading {
    font-size: 35px;
  }
}
