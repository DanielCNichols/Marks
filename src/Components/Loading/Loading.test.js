import React from 'react';
import ReactDOM from 'react-dom';
import Loading from './Loading';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe('Loading component testing', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(
      <MemoryRouter>
        <Loading />
      </MemoryRouter>,
      div
    );

    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Loading />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
