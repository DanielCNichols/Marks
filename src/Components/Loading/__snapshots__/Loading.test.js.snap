// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component testing renders UI as expected 1`] = `
<div
  className="loading"
>
  <p>
    Loading...
  </p>
</div>
`;
